### PR TITLE
[Polymer][Face] Rename manifest.json.bak to manifest.json

### PR DESCRIPTION
**Pull request by halton**

### CODE REVIEW

**Comment by halton** [comment]
@huningxin @leonhsl PTAL

**Comment by leonhsl** [comment]
lgtm and Thanks!

File: sample/src_polymer/manifest.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-  "xwalk_extensions": [
-    "realsense_extensions/enhanced_photography",
-    "realsense_extensions/face",
-    "realsense_extensions/scene_perception"]
-}
-
+{
+  "name": "Crosswalk RealSense Sample App - Polymer",
+  "start_url": "index.html",
+  "xwalk_package_id": "org.crosswalk_project.rs_sample",
```
Comment (huningxin) [inline sample/src_polymer/manifest.json]: Would this make a name conflict with old sample? Or we add polymer into it "org.corsswalk_project.rs_sample_polymer"?

**Comment by halton** [comment]
@huningxin good catch, I'd suggest `org.crosswalk_project.rs_sample_polymer` ? In future, we remove old UI, we can change it back.  WDYT?

**Comment by huningxin** [comment]
Your proposal sounds good to me.

**Comment by halton** [comment]
@huningxin okay to merge?

**Comment by huningxin** [comment]
LGTM.

**Pull request merged**